dev/git.ts

```diff
@@ -171,10 +171,21 @@ const gitGenerators: Record<string, Fig.Generator> = {
       }
 
       const files = output.split("\n").map((file) => {
+        // From "git status --short"
+        // M  dev/github.ts // test file that was added
+        //  M dev/kubectl.ts // test file that was not added
+        // A  test2.txt // new added and tracked file
+        // ?? test.txt // new untracked file
+        const alreadyAdded = file.charAt(0) === "M" || file.charAt(0) === "A";
```

Comment:
This might be a bit overkill but we can simplify it with:
```suggestion
        const alreadyAdded = ["M", "A"].includes(file.charAt(0));
```

Reply:
You're making Linus Torvalds proud! 😄